.env

```diff
@@ -0,0 +1,14 @@
+DEBUG=on
+DATABASE_NAME=oss_challenge
+DATABASE_USER=osschallenge
+DATABASE_PASSWORD=osschallenge
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+EMAIL_HOST_USER=
+DEFAULT_FROM_EMAIL=osschallenge@osschallenge.com
+EMAIL_HOST_PASSWORD=
+EMAIL_HOST=mailcatcher
+EMAIL_PORT=1025
+EMAIL_USE_TLS=False
+SECRET_KEY^0)(ucw7sr6qb2q%%13)f1&f_qmom#do*soro00@!00agrnwta
```

Comment:
There's no `=` here, this variable assignment doesn't work

Reply:
fixed